### PR TITLE
Hamburger menu

### PR DESCRIPTION
**Pull request by Abe1able**
### In this pull request, we've used pair programming to

- create a hamburger menu
- use javascript to add interaction to the hamburger button
- add styling to the menu.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,18 +15,23 @@
...
+          <li class="nav-item"><a class="nav-link" href="#work-section">Portfolio</a></li>
+          <li class="nav-item"><a class="nav-link" href="#about-section">About</a></li>
+          <li class="nav-item"><a class="nav-link" href="#contact-form">Contact</a></li>
         </ul>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
       </nav>
```
Comment (Nemwel-Boniface) [inline index.html]: - [x] Great job implementing the mobile menu :clap: / I have realized that there is a white space at the top of the menu when it opens. Kindly fix this to make your mobile menu perfect!

**This is how I am seeing it on my end:**

image


**This is the desired design:**

image

File: index.html [hunk 2]
```diff
@@ -15,18 +15,23 @@
...
+          class="only-mobile hamburger-btn hamburger"
           data-mobile-menu-target="#mobile-menu"
         >
           <img src="Union.svg" alt="Hamburger Navbar Icon" />
-        </button>
-        <ul class="only-desktop menu-bar">
-          <li><a href="#work-section">Portfolio</a></li>
-          <li><a href="#about-section">About</a></li>
-          <li><a href="#contact-form">Contact</a></li>
+        </button> -->
```
Comment (Nemwel-Boniface) [inline index.html]: - [x] Kindly remove all commented out and unused/unwanted pieces of code from your code base. It make it look messy and cluttered.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Abe1able            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job with your portfolio mobile menu so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Abe1able) [reply index.html]: Removed commented out pieces of code. Thank you.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Abe1able , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**